config/quark.cfg

```diff
@@ -0,0 +1,3190 @@
+            minecraft:white_shulker_box
+            minecraft:orange_shulker_box
+            minecraft:magenta_shulker_box
+            minecraft:light_blue_shulker_box
+            minecraft:yellow_shulker_box
+            minecraft:lime_shulker_box
+            minecraft:pink_shulker_box
+            minecraft:gray_shulker_box
+            minecraft:silver_shulker_box
+            minecraft:cyan_shulker_box
+            minecraft:purple_shulker_box
+            minecraft:blue_shulker_box
+            minecraft:brown_shulker_box
+            minecraft:green_shulker_box
+            minecraft:red_shulker_box
+            minecraft:black_shulker_box
+         >
+    }
+
+    "look down on ladders to descend fast" {
+        # Should your character automatically sneak on ladders in GUIs?
+        B:"Sneak While In Gui"=true
+    }
+
+    "pat the dogs" {
+        # How many ticks it takes for a dog to want affection after being pet/tamed; leave -1 to disable
+        I:"Dog loneliness"=-1
+    }
+
+    "improved sleeping" {
+        B:"Enable AFK"=true
+
+        # This feature disables itself if any of the following mods are loaded: 
+        #  - morpheus
+        #  - sleepingoverhaul
+        # This is done to prevent content overlap.
+        # You can turn this on to force the feature to be loaded even if the above mods are also loaded.
+        B:"Force Enabled"=false
+
+        # The percentage of the (non-afk) server that needs to be sleeping for the time to change.
+        I:"Required Percentage"=100
+
+        # How many ticks are required for a player to be marked AFK
+        I:"Time for AFK"=2400
+    }
+
+    "stackable items" {
+        # The format for setting an item's max size is item name|stacksize. i.e. `minecraft:saddle|8`
+        S:"Stackable Items" <
+            minecraft:minecart|16
+            minecraft:chest_minecart|16
+            minecraft:command_block_minecart|16
+            minecraft:furnace_minecart|16
+            minecraft:hopper_minecart|16
+            minecraft:tnt_minecart|16
+            minecraft:mushroom_stew|64
+            minecraft:rabbit_stew|64
+            minecraft:beetroot_soup|64
+            minecraft:saddle|8
+         >
+    }
+
+    "right click harvest" {
+        # Should Quark look for (nonvanilla) crops, and handle them?
+        B:"Add Harvestable Crops"=false
+
+        # Can players harvest crops with empty hand clicks?
+        B:"Empty Hand Harvest"=true
+
+        # This feature disables itself if any of the following mods are loaded: 
+        #  - harvest
+        # This is done to prevent content overlap.
+        # You can turn this on to force the feature to be loaded even if the above mods are also loaded.
+        B:"Force Enabled"=false
+
+        # Which crops can be harvested?
+        # Format is: "harvestState[,afterHarvest]", i.e. "minecraft:wheat:7" or "minecraft:cocoa:11,minecraft:cocoa:3"
+        S:"Harvestable Crops" <            
```

Comment:
What about the other crops added by "Unique Crops"?

Reply:
Intentionally not included, see pull request description.